applications/FluidDynamicsApplication/python_scripts/navier_stokes_two_fluids_solver.py

```diff
@@ -95,6 +97,15 @@ def __init__(self, model, custom_settings):
 
         self._bfecc_convection = self.settings["bfecc_convection"].GetBool()
 
+        self._eulerian_error_compensation = False
+        if (self.settings.Has("eulerian_error_compensation")):
```

Comment:
I think the has check is not required (it will always be true after the validate & assign defaults).

Reply:
So you mean if the input settings does not have "eulerian_error_compensation", the default value should have been assigned automatically. Do I need to add anything else in this regard or "eulerian_error_compensation" : false," in default_settings is enough?

Reply:
Exactly. If the input settings doesn't have `eulerian_error_compensation` they will be assigned to false (OFC provided that we add this to the default settings, which is the standard mechanism for these things).

Reply:
So, I assume that I can remove the extra lines for
"eulerian_error_compensation"
"split_levelset"
"surface_tension"
"distance_reinitialization"
and
"distance_smoothing"
since all of them are already mentioned in GetDefaultParameters.

Reply:
If these are in the defaults yes, you can directly access the value (which would be either the user-defined one or the one in the defaults json string).